praetorian_cli/sdk/keychain.py

```
@@ -119,6 +120,13 @@ def base_url(self):
         """ Get the base URL for the backend. It is the "api" field in the keychain file. """
         return self.get_option('api')
 
+    def websocket_url(self):
+        """ Get the optional WebSocket endpoint URL (profile 'websocket' option or PRAETORIAN_CLI_WS_URL env). Returns None if unset. """
+        try:
+            return self.get_option('websocket')
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)